### PR TITLE
install: add callout to next steps in terminal display

### PR DESCRIPTION
**Pull request by alixander**
@ejulio-ts missed that he had to do something after install, and I think it won't be uncommon for others to as well. This adds a little callout that should catch the majority of cases.

Also adds a link to report issue if something went wrong

## Before

Screen Shot 2022-11-17 at 11 24 10 AM

## After

Screen Shot 2022-11-17 at 11 30 53 AM

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -214,7 +214,7 @@ hide() {
     return
   fi
   cat "$out" >&2
-  exit "$code"
+  return "$code"
```
Comment (alixander) [inline install.sh:217]: i didn't change these, just wasn't `gen`d before

**Review by nhooyr — APPROVED** [review]
✅

File: ci/release/_install.sh [hunk 2]
```diff
@@ -318,7 +318,7 @@ uninstall_standalone_d2() {
 
   if [ ! -e "$INSTALL_DIR/d2-$INSTALLED_VERSION" ]; then
     warn "missing standalone install release directory $INSTALL_DIR/d2-$INSTALLED_VERSION"
-    warn "d2 have been installed via some other installation method."
+    warn "d2 has been installed via some other installation method."
```
Comment (nhooyr) [inline ci/release/_install.sh:321]: must have

**Review by nhooyr — CHANGES_REQUESTED** [review]

**Review by nhooyr — APPROVED** [review]

**Review by ejulio-ts — APPROVED** [review]
😅

**Pull request merged**